### PR TITLE
support coredns v1.11.3

### PR DESCRIPTION
**Pull request by dims**
started with https://github.com/kubernetes/k8s.io/pull/7086 and realized we need to update here as well.

cc @chrisohaver @johnbelamaric

### CODE REVIEW

**Review by johnbelamaric — APPROVED** [review]

File: migration/versions.go [hunk 1]
```diff
@@ -30,6 +30,11 @@ type release struct {
 
 // Versions holds a map of plugin/option migrations per CoreDNS release (since 1.1.4)
 var Versions = map[string]release{
+	"1.11.3": {
+		priorVersion:   "1.11.1",
+		dockerImageSHA: "9caabbf6238b189a65d0d6e6ac138de60d6a1c419e5a341fbbb7c78382559c6e",
+		plugins: 		plugins_1_11_0,
+	},
 	"1.11.1": {
 		priorVersion:   "1.11.0",
```
Comment (johnbelamaric) [inline migration/versions.go:39]: @dims @be the elder I think we must need a "next version" here.

Comment (dims) [reply migration/versions.go:39]: adding a PR gimme a sec

**Pull request merged**